vnext/JSI/Shared/ChakraObjectRef.cpp

```diff
@@ -128,13 +129,17 @@ ChakraObjectRef GetPropertyId(const std::string_view &utf8) {
     throw facebook::jsi::JSINativeException("Property name cannot be a nullptr.");
   }
 
-  // We use a #ifdef here because we can avoid a UTF-8 to UTF-16 conversion
-  // using ChakraCore's JsCreatePropertyId API.
+// We use a #ifdef here because we can avoid a UTF-8 to UTF-16 conversion
+// using ChakraCore's JsCreatePropertyId API.
 #ifdef CHAKRACORE
-  JsPropertyIdRef id = JS_INVALID_REFERENCE;
-  VerifyChakraErrorElseThrow(JsCreatePropertyId(utf8.data(), utf8.length(), &id));
-  return ChakraObjectRef(id);
-
+  if (!Microsoft::React::GetRuntimeOptionBool("ForceSystemChakra")) {
```

Comment:
>if (!Microsoft::React::GetRuntimeOptionBool("ForceSystemChakra")) { [](start = 2, length = 67)

Is it to enable the FeatureGate? I.e. it is the reason that we completely did not remove the whole #ifdef, right?

Reply:
Correct. The conditional compilation is still necessary in many spots.

The general idea is:
- Be able to compile RNW Desktop against ChakraCore includes and import libs.
- Be able to swap from ChakraCore to System Chakra at runtime.
- Leave MSRN (UWP) intact.